nix/bitstream.nix

```diff
@@ -0,0 +1,26 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+{
+  pkgs,
+  lrPkgs,
+  pythonEnv,
+}: {
+  build = pkgs.writeShellApplication {
```

Comment:
You need https://github.com/lowRISC/sonata-system/blob/6ceaa9c298e321eba679d71388da0d6d3e8cd187/flake.nix#L146-L147

Reply:
Why this is needed? `writeShellApplication` does not have these arguments like derivation.

Reply:
These are passed as environment variables into fusesoc

Reply:
fixed.